src/wallet/mod.rs

```diff
@@ -52,7 +52,10 @@ pub use utils::IsDust;
 
 #[allow(deprecated)]
 use address_validator::AddressValidator;
-use coin_selection::DefaultCoinSelectionAlgorithm;
+use coin_selection::{
+    get_selection, DefaultCoinSelectionAlgorithm,
+    Excess::{Change, NoChange},
```

Comment:
> Naming request: A name like `HandleExcess` could be more clearer here.. Its a simple enum, that specifies how to handle the change if any.

Well, I thought about that before and didn't like it because `Excess` is just a "thing" and not an action per se. The logic inside `create_tx` is the one that handles the `Excess`. That was my reasoning. If I didn't convince you, let me know and I will change it.

> Not a major issue, a general rust street rule is not to import enum variants directly. Call them with the enum name in code like `Excess::Change` etc. The name of the enum gives useful context at the usage location. And its always just two item long.

Good rule! I'll apply it.